getstartedwithSDK/2tracingWorkflow.mdx

```diff
@@ -1,23 +1,22 @@
 ---
-title: "Tracing an LLM Chain"
-sidebarTitle: Tracing an LLM Chain
+title: "Get Started with Tracing Multi-Step LLM Workflows"
+sidebarTitle: Tracing Multi-Step LLM Workflows
 description: "Get started with tracing your LLM Chain with Baserun within 3 minutes."
 ---
-
-![Trace a LLM chain](/images/trace_detail.png)
-
 ## Introduction:
 
 A Trace comprises a series of events executed within an LLM chain. Tracing enables Baserun to capture and display the LLM chain's entire lifecycle, whether synchronous or asynchronous.
 
 Tracing LLM chains allows you to debug your application, monitor your LLM chains' performance, and collect customer feedback.
 
+
+![Trace a LLM chain](/images/trace_detail.png)
+
 ## Use case:
 
 Please reference the [Logging Overview](/logging/overview) to learn why logging LLM chain is critical for LLM feature development.
 
 ## Features:
-
 - Support Python and Typescript
 - Automatically logs OpenAI and Anthropic LLM calls
 - UI to show sequence of events
```

Comment:
* Support Python and Typescript
* Automatically logs OpenAI and Anthropic LLM calls
* UI to show sequence of events
* Provide generative token usage, context token usage, estimated cost, duration
* Support automatic evaluation
* Supports async functions
* Option to add custom trace name (See advanced tracing feature)
* Option to add custom metadata (See advanced tracing feature)
* Option to set trace result (See advanced tracing feature)
* Support collecting user feedback